core/metacling/src/TCling.cxx

```diff
@@ -6157,7 +6167,7 @@ static void* LazyFunctionCreatorAutoloadForModule(const std::string& mangled_nam
             llvm::StringRef dContents;
             section.getContents(dContents);
             // If this library contains mangled name
-            if (dContents.contains(mangled_name)) {
+            if (dContents.contains(name_in_so)) {
```

Comment:
@yamaguchi1024, could you review this particular change, please?

Reply:
Answering to myself: this should be indifferent because we check if the section `contains` the mangled name. I will revert it to what it was.